main.py

```diff
@@ -1,10 +1,49 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        players = json.load(file)
+
+    for player, player_data in players.items():
+        player_race = player_data["race"]
+        player_skills = player_race["skills"]
+        player_guild = player_data["guild"]
+
+        race, _ = Race.objects.get_or_create(
+            name=player_race["name"],
+            defaults={"description": player_race["description"]}
+        )
+
+        for skill_data in player_skills:
+            Skill.objects.get_or_create(
+                ame=skill_data["name"],
+                defaults={
+                    "bonus": skill_data["bonus"],
+                    "race": race
+                }
+            )
+
+            guild = None
+            if player_guild:
+                guild, _ = Guild.objects.get_or_create(
+                    name=player_guild["name"],
+                    defaults={"description": player_guild["description"]}
+                )
```

Comment:
The logic for creating or retrieving a guild should be placed outside the skills loop to avoid unnecessary repeated operations. Consider moving this block of code before the skills loop.